pyocd/utility/cmdline.py

```diff
@@ -161,7 +161,7 @@ def convert_one_session_option(name: str, value: Optional[str]) -> Tuple[str, An
     result = None
 
     # Extract the option's type. If its type is a tuple of types, then take the first type.
-    if info.type is tuple:
+    if typle(info.type) is tuple:
```

Comment:
`type()`? 😄 

Anyway, using `isinstance()` is recommended instead of `type()` and an "is" comparison. That way it works for subclasses (though probably not a concern for this use case).